care/facility/api/serializers/prescription.py

```diff
@@ -57,6 +62,25 @@ def validate(self, attrs):
                 MedibaseMedicine, external_id=attrs["medicine"]
             )
 
+        if not self.instance:
+            consultation_obj = get_object_or_404(
+                PatientConsultation,
+                external_id=self.context["request"].parser_context["kwargs"][
+                    "consultation_external_id"
+                ],
+            )
+
+            if Prescription.objects.filter(
+                consultation=consultation_obj,
```

Comment:
Can we not query the Prescription object directly via `consultation__external_id=self.context["request"].parser_context["kwargs"]["consultation_external_id"]`? That'd save a db query fetching the PatientConsultation object.

Reply:
Done!